src/components/Dropdowns/YearDropdown/index.js

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { MenuItem, TextField } from '@material-ui/core';
+import { getIn } from 'formik';
+
+const yearDropdownChoices = (minYear, maxYear) => {
+  const yearChoices = [];
+
+  for (let year = 0; year <= maxYear - minYear; year += 1) {
+    yearChoices.push(year + minYear);
+  }
+
+  return yearChoices;
+};
+
+/*
+ * From https://github.com/stackworx/formik-material-ui/tree/master/packages/formik-material-ui/src (TextField) :p
+ * This essentially takes the props received at the top-level <Field /> as well
+ * as the default props for that <Field /> (field, form, meta) and transforms
+ * all that into a props that is suitable for MUI's <TextField />
+ */
+export function fieldToTextField({
+  disabled,
+  field,
+  form: { isSubmitting, touched, errors },
+  ...props
+}) {
+  const fieldError = getIn(errors, field.name);
+  const showError = getIn(touched, field.name) && !!fieldError;
+
+  return {
+    ...props,
+    ...field,
+    error: showError,
+    helperText: showError ? fieldError : props.helperText,
+    disabled: disabled ?? isSubmitting,
```

Comment:
again change to null check for future sustainability - not a big fan of ??

Reply:
Honestly that stuff is the stuff that came from formik-material-ui that I didn't touch. I'll change it tho